### PR TITLE
minimum viable change to enable delegated managers

### PR DESCRIPTION
**Pull request by ozzyjohnson**
As stated in the comments...

    // Needed a way to allow OrgManagers to set and unset space roles
    // as the UAA call here fails for non-admins. Seems like this would
    // be better handled with a /v2/users that returns a combined collection 
    // of users for the caller's orgs, but this works.

TODO: Tests.

### CODE REVIEW

**Comment by ozzyjohnson** [comment]
Wrong target.

File: cf/api/users.go [hunk 1]
```diff
@@ -58,28 +59,76 @@ func NewCloudControllerUserRepository(config core_config.Reader, uaaGateway net.
...
 	var user models.UserFields
+
 	if apiErr != nil {
 		return user, apiErr
 	}
 
-	usernameFilter := neturl.QueryEscape(fmt.Sprintf(`userName Eq "%s"`, username))
-	path := fmt.Sprintf("%s/Users?attributes=id,userName&filter=%s", uaaEndpoint, usernameFilter)
-	users, apiErr := repo.updateOrFindUsersWithUAAPath([]models.UserFields{}, path)
+	// var myGUID string
```
Comment (maximilien) [inline cf/api/users.go]: Why are we leaving what seems to be debugging code?

File: cf/api/users.go [hunk 2]
```diff
@@ -58,28 +59,76 @@ func NewCloudControllerUserRepository(config core_config.Reader, uaaGateway net.
...
+			return user, errors.NewModelNotFoundError("User", username)
+		}
+
+		return users[0], apiErr
 	}
+	return user, apiErr
+}
 
-	return users[0], apiErr
+func (repo CloudControllerUserRepository) ListUsersInOrgWithNoUAA(orgGuid string) (users []models.UserFields, apiErr error) {
```
Comment (maximilien) [inline cf/api/users.go]: Where is the test for this public method?

File: cf/api/users.go [hunk 3]
```diff
@@ -58,28 +59,76 @@ func NewCloudControllerUserRepository(config core_config.Reader, uaaGateway net.
 
 func (repo CloudControllerUserRepository) FindByUsername(username string) (models.UserFields, error) {
```
Comment (maximilien) [inline cf/api/users.go]: Where is the test for this?

**Pull request closed without merging**